### ISSUE TITLE
emoji support

### ISSUE THREAD

**Issue by zbeekman** [FabricLabs/soundtrack#103]
I feel like this would be a relatively easy change to make… but I no haz JS skillz :+1: 

https://github.com/hassankhan/emojify.js :boom:

**Comment by zbeekman** [comment]
@naterchrdsn has this ever been merged? It looks like maybe this is sufficient?

**Comment by naterchrdsn** [comment]
nah, holding off until maki is fleshed out a bit more then ill add this feature
